rest-api/controllers/pokemon_feed.go

```diff
@@ -31,6 +32,9 @@ type ApiEndpointBody struct {
 	} `json:"results"`
 }
 
+var total int
+var total_per_worker [3]int
```

Comment:
If you have a spare time, a good resource to know about the Go style is under Effective Go.
https://golang.org/doc/effective_go#mixed-caps

Under Go is preferable to use MixedCaps instead of snake_case :)

Reply:
great, I made a few updates on this file and i will keep this in mind